examples/AgraphExample.py

```diff
@@ -0,0 +1,73 @@
+# Ignoring some linting rules in tests
+# pylint: disable=redefined-outer-name
+# pylint: disable=missing-docstring
+import numpy as np
+
+from bingo.SymbolicRegression.AGraph.AGraphCrossover import AGraphCrossover
+from bingo.SymbolicRegression.AGraph.AgraphMutation import AGraphMutation
+from bingo.SymbolicRegression.AGraph.AGraphGenerator import AGraphGenerator
+from bingo.SymbolicRegression.AGraph.ComponentGenerator import ComponentGenerator
+from bingo.SymbolicRegression.ExplicitRegression import ExplicitRegression, ExplicitTrainingData
+
+from bingo.Base.AgeFitnessEA import AgeFitnessEA
+from bingo.Base.Evaluation import Evaluation
+from bingo.Base.Island import Island
+from bingo.Base.ContinuousLocalOptimization import ContinuousLocalOptimization
+
+POP_SIZE = 100
+STACK_SIZE = 10
+
+def init_x_vals(start, stop, num_points):
+    return np.linspace(start, stop, num_points).reshape([-1, 1])
+
+def equation_eval(x):
+    return x**2 + 3.5*x**3
+
+def execute_generational_steps():
+    x = init_x_vals(-10, 10, 100)
+    y = equation_eval(x)
+    training_data = ExplicitTrainingData(x, y)
+
+    component_generator = ComponentGenerator(x.shape[1])
+    component_generator.add_operator(2)
+    component_generator.add_operator(3)
+    component_generator.add_operator(4)
+
+    crossover = AGraphCrossover()
+    mutation = AGraphMutation(component_generator)
+
+    agraph_generator = AGraphGenerator(STACK_SIZE, component_generator)
+
+    fitness = ExplicitRegression(training_data=training_data)
+    local_opt_fitness = ContinuousLocalOptimization(fitness, algorithm='lm')
+    evaluator = SimpleEvaluation(local_opt_fitness)
+
+    ea = AgeFitnessEA(evaluator, agraph_generator, crossover,
+                      mutation, 0.4, 0.4, POP_SIZE)
+
+    island = Island(ea, agraph_generator, POP_SIZE)
+
+    curr_fitness = 10e6
```

Comment:
I updated so you can now run best individual at age 0,  lets use the real value of best fitness here